processing/src/main/java/org/apache/carbondata/processing/loading/sort/impl/ParallelReadMergeSorterImpl.java

```diff
@@ -154,11 +155,13 @@ public void close() {
   /**
    * Below method will be used to process data to next step
    */
-  private boolean processRowToNextStep(SortDataRows sortDataRows, SortParameters parameters)
+  private boolean processRowToNextStep(SortDataRows[] sortDataRows, SortParameters parameters)
```

Comment:
i think there duplicate code fragments in unsafe and safe, please check and better to refactor.

Reply:
just modify one parameter from origin code

Reply:
better to refactor when we have chance.